lib/mm_network.pm

```diff
@@ -79,10 +79,12 @@ sub configure_default_gateway {
 
 sub configure_static_dns {
     my ($conf) = @_;
-    my $servers = join(" ", @{$conf->{nameserver}});
-    script_run("sed -i -e 's|^NETCONFIG_DNS_STATIC_SERVERS=.*|NETCONFIG_DNS_STATIC_SERVERS=\"$servers\"|' /etc/sysconfig/network/config");
-    script_run("netconfig -f update");
-    script_run("cat /etc/resolv.conf");
+    my @servers = $conf->{nameserver};
+    assert_script_run("ping -c 1 $_") foreach @servers;
```

Comment:
@okurz it tries to ping an array: https://openqa.opensuse.org/tests/995903#step/before_test/45